webapp/src/components/admin_settings/jitsi_settings.tsx

```diff
@@ -0,0 +1,300 @@
+import * as React from 'react';
+import {FormattedMessage} from 'react-intl';
+import JaaSSection from './jaas_section';
+import {JitsiSection, JITSI_NAMING_SCHEME} from './jitsi_section';
+import {id as pluginId} from '../../manifest';
+import I18nProvider from 'components/i18n_provider';
+
+type Props = {
+    id: string,
+    label: string,
+    value: any,
+    disabled: boolean,
+    config: any,
+    license: object,
+    setByEnv: boolean,
+    onChange: Function,
+    registerSaveAction: Function,
+    setSaveNeeded: Function,
+    unRegisterSaveAction: Function
+};
+
+type Settings = {
+    usejaas?: boolean,
+    jitsiurl?: string,
+    jitsiembedded?: boolean,
+    jitsinamingscheme?: string,
+    jitsijwt?: boolean,
+    jitsiappid?: string,
+    jitsiappsecret?: string,
+    jitsilinkvalidtime?: number,
+    jitsicompatibilitymode?: boolean,
+    jaasappid?: string,
+    jaasapikey?: string,
+    jaasprivatekey?: string
```

Comment:
Change the casing to make it consistent with the other variables.

Reply:
discussed